src/app/Main/Body/InfoPage/InfoPage-Container.component.tsx

```diff
@@ -0,0 +1,333 @@
+        "1",
+        "128"
+    ),
+];
+
+const tableAmrMrsaRows: AmrsTableData[] = [
+    createAmrTableOneCutOff("Aminoglykoside", "Gentamicin", "2", "1", "16"),
+    createAmrTableOneCutOff("Aminoglykoside", "Kanamycin", "8", "4", "64"),
+    createAmrTableOneCutOff("Aminoglykoside", "Streptomycin", "16", "4", "32"),
+    createAmrTableOneCutOff("Amphenicole", "Chloramphenicol", "16", "4", "64"),
+    createAmrTableOneCutOff(
+        "Fluorchinolone",
+        "Ciprofloxacin",
+        "1",
+        "0,25",
+        "8"
+    ),
+    createAmrTableOneCutOff("Penicilline", "Penicillin G", "0,12", "0,12", "2"),
+    createAmrTableOneCutOff("Cephalosporine", "Cefoxitin", "4", "0,5", "16"),
+    createAmrTableOneCutOff(
+        "Folatsynthesehemmer",
+        "Trimethoprim",
+        "2",
+        "2",
+        "32"
+    ),
+    createAmrTableOneCutOff(
+        "Sulfonamide",
+        "Sulfamethoxazol",
+        "128",
+        "64",
+        "512"
+    ),
+    createAmrTableOneCutOff("Tetrazykline", "Tetrazyklin", "1", "0,5", "16"),
+    createAmrTableOneCutOff("Lincosamide", "Clindamycin", "0,25", "0,12", "4"),
+    createAmrTableOneCutOff("Makrolide", "Erythromycin", "1", "0,25", "8"),
+    createAmrTableOneCutOff(
+        "Pseudomonische Säuren",
+        "Mupirocin",
+        "1",
+        "0,5",
+        "256"
+    ),
+    createAmrTableOneCutOff("Ansamycine", "Rifampicin", "0,03", "0,016", "0,5"),
+    createAmrTableOneCutOff("Oxazolidinone", "Linezolid", "4", "1", "8"),
+    createAmrTableOneCutOff(
+        "Triterpensäuren",
+        "Fusidinsäure",
+        "0,5",
+        "0,5",
+        "4"
+    ),
+    createAmrTableOneCutOff(
+        "Streptogramine",
+        "Quinupristin / Dalfopristin",
+        "1",
+        "0,5",
+        "4"
+    ),
+    createAmrTableOneCutOff("Pleuromutiline", "Tiamulin", "2", "0,5", "4"),
+    createAmrTableOneCutOff("Glykopeptide", "Vancomycin", "2", "1", "16"),
+];
+
+const tableAmrEfRows: AmrsTableData[] = [
+    createAmrTable("Aminoglykoside", "Gentamicin", "32", "32", "8", "1024"),
+    createAmrTable("Amphenicole", "Chloramphenicol", "32", "32", "4", "128"),
+    createAmrTable("Fluorchinolone", "Ciprofloxacin", "4", "4", "0,12", "16"),
+    createAmrTable("Aminopenicilline", "Ampicillin", "4", "4", "0,5", "64"),
+    createAmrTable("Tetrazykline", "Tetrazyklin", "4", "4", "1", "128"),
+    createAmrTable("Makrolide", "Erythromycin", "4", "4", "1", "128"),
+    createAmrTable("Lipopeptide", "Daptomycin", "4", "4", "0,25", "32"),
+    createAmrTable("Oxazolidinone", "Linezolid", "4", "4", "0,5", "64"),
+    createAmrTable("Glycilcycline", "Tigecyclin", "0,25", "0,25", "0,03", "4"),
+    createAmrTable("Glykopeptide", "Teicoplanin", "2", "2", "0,5", "64"),
+    createAmrTable("Glykopeptide", "Vancomycin", "4", "4", "1", "128"),
+];
+
+export function InfoPageContainerComponent(): JSX.Element {
+    const { t } = useTranslation(["InfoPage"]);
+
+    const describedFilters = [

```

Comment:
So far there are no plans for an api. My thought was to put all the variables that depend on the data into the conatiner.

Reply:
Ok. You can leave it here if you like it. Personally i would define the data where i use it (former filterAccordionsComponent), if the container does not need to manipulate or load it. Another solution could be to put the described filters also to a constants file. Or yet another solution would be a react context object that holds all the constants of your app and is used like the translation context.
PS: So your solution says: the InfoPage component is configurable by setting which described filters it should display. But then this generalization is only partly done because the filteAccordions component does some calculations with the values behind the described filters.